content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/configure-limit-parsys/touchui-limit-parsys.js

```diff
@@ -34,6 +34,15 @@
     function _findPropertyFromPolicy(editable, design, propertyName) {
         var cell = Granite.author.util.resolveProperty(design, editable.config.policyPath);
 
+        if (!cell || !cell[propertyName]) {
+            // Inherit property also from its parent (if not set in the local policy path)
+            var parent = ns.editables.getParent(editable);
```

Comment:
we might need to change "nc" for Granite.author. I don't see where it is defined

Reply:
good catch, fixed